### ISSUE TITLE
Adding files to release archive at root of the project

### ISSUE THREAD

**Issue by gmalkas** [bitwalker/exrm#104]
Hey,

I'm trying to add non-Elixir files to my release. For instance, I need to include a shell script (`script/start_server`) in the release archive (the `script` folder should be at the same level than the `bin` folder ideally) but I'm not really sure how to do this with exrm.

Is there a configuration file where I could specify these kind of things?

Thanks and sorry if that's the wrong place to ask :D

**Comment by bitwalker** [comment]
@gmalkas Hey no worries, there is actually a bug with this (my fault) which I need to address in order for the standard `relx.config` overlays to work. I'm going to treat this as a bug report for the moment and try and get it addressed by this weekend. Sorry for the trouble!

**Comment by bitwalker** [comment]
Ok @gmalkas I just merged a temporary fix for this that brings back support for this, you can use the following relx.config file (by placing it in `<project root>/rel/relx.config`) to copy the file you mentioned into the release tar.

```erlang
{overlay, [{copy, "../../script/start_server", "script/start_server"}]}.
```

The odd relative path for the first reference is a bug I still need to fix, because the config is actually merged into the default one stored in a different location, and when that config is loaded the paths are relative to the exrm config file, not yours, which isn't what we want. Anyway, the path you should be building relative paths from is `<project root>/rel/files/relx.config`, so if you have a `script` folder in your project root, you'll refer to it with `../../script` in the `relx.config` you place in `rel/relx.config`. A headache to remember, but I'll be fixing this soon.

**Comment by bitwalker** [comment]
Oh, and this fix is in 0.15.1 by the way.

**Comment by gmalkas** [comment]
It worked! Thanks a lot :+1:
